README.md

```diff
@@ -30,6 +30,27 @@ or using the `install` target in the `Makefile` by simply running
     make install
 
 
+## Computing p-values
```

Comment:
I'd add a blank line here. We do that for every second-level header in the rest of the readme.

Reply:
Ok, did it.